test/views/updateRegisteredDetails/WarehouseDetailsViewSpec.scala

```diff
@@ -0,0 +1,231 @@
+/*
```

Comment:
this view test is missing two tests
1. when I pass in None for summary list, what is the content
2. when I pass in Some(summary list) what is the content

Reply:
it shows this message
![image](https://github.com/hmrc/soft-drinks-industry-levy-variations-frontend/assets/75027964/c4049326-961a-45dd-94dc-1caa10e8ee1f)